### DISCUSSION TITLE
How to lock model position in AR

### DISCUSSION THREAD

**Discussion by barzorke** [google/model-viewer#2991] [Q&A]
Hi,

I am trying to lock the AR position/rotation once placed on the ground.

Thanks for any guidance

For now, I did try with these :
https://github.com/google/model-viewer/discussions/2120
https://modelviewer.dev/examples/scenegraph/#transforms

**Comment by samaneh-kazemi** [comment]
Hi @barzorke, can you share more details on how you're using this?

**Comment by elalish** [comment]
We don't  support  that currently; can  you explain your use case of why it would be desired for the  user  to not be able to control where the object is? AR is finicky and  often  the initial placement  is not really correct.

**Reply by barzorke** [reply]
I would like a calibration phase where the user place the object once, then it would remain in place forever and ever.

**Reply by elalish** [reply]
Well, that might be nice if  it were  possible, but as I said, AR just isn't that  reliable. Giving the  user no recourse to change the placement when it's wrong gives a pretty bad  experience.
